Traceback (most recent call last):
  File "/home/hz466/calendar-planning/source/../output/SMT/DeepSeek-R1/meeting/code/meeting_planning_example_763.py", line 153, in <module>
    main()
  File "/home/hz466/calendar-planning/source/../output/SMT/DeepSeek-R1/meeting/code/meeting_planning_example_763.py", line 116, in main
    other_met, other_start, other_end, other_loc = friend_vars[other['name']]
                                                   ~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: 'Mark'
